long-term-storage-aws.md

```diff
@@ -31,14 +51,20 @@ config:
 ```
 **Note:** given that this is yaml, it requires this specific indention.
 
-Instead of using a service key, you can alternatively attach the policy to the Thanos pods service accounts. Your `object-store.yaml` should follow the format below when using this option, which does not contain the secret_key and access_key fields.
+## <a name="attach-role"></a>Attach IAM role to Service Account Method
+
+Instead of using a secret key in a file, many will want to use this method.
+
+Attach the policy to the Thanos pods service accounts. Your `object-store.yaml` should follow the format below when using this option, which does not contain the secret_key and access_key fields.
 
 ```
 type: S3
 config:
   bucket: "kc-thanos-store"
   endpoint: "s3.amazonaws.com"
   region: "us-east-1"
+  # access_key: "AKIAXW6UVLRRTDSCCU4D"
```

Comment:
Let's replace the access key with something similar to what was done for secret-key